src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-weight: 300;
+  font-family: Avenir;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-weight: 800;
+  font-family: Avenir;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  box-sizing: border-box;
```

Comment:
apply `box-sizing` to the all elements or remove it. It is confusing when there is a difference in size calculationg approach